cleanlab/filter.py

```diff
@@ -138,6 +139,9 @@ class 0, 1, ..., K-1.
       list of labels for each example, instead of just a single label.
       The multi-label setting supports classification tasks where an example has 1 or more labels.
       Example of a multi-labeled `labels` input: ``[[0,1], [1], [0,2], [0,1,2], [0], [1], ...]``.
+      Confident joint should be an array of shape ``(N, K, K)``
```

Comment:
```suggestion
      Confident joint should be an array of shape ``(K, 2, 2)`` in multi-label settings.
```

Reply:
Note throughout this package, we reserve `N` for number of examples, `K` for number of classes.